app/assets/javascripts/game_moves.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
+  $( ".piece_image" ).draggable();
+  $( "td" ).droppable({
+    
+    drop: function( event, ui ) {
+     var game_id = $('#board').data('game_id');
+
+      $.ajax( {  type: 'PUT',
+        url: '/games/' + game_id + '/move',
+        dataType: 'json',
+
+        data: {
+          position_x: $(event.target).data('row'),
+          position_y: $(event.target).data('col'),
+          piece_id: $(ui.draggable).data("piece_id")
+        }
+      })
+      .success(function(data){
+       $(ui.draggable).detach().css({top: 0,left: 0}).appendTo(this);

```

Comment:
what does the {top: 0,left: 0} portion of this code accomplish?

Reply:
This is the Snap-To-Middle functionality, so it will always center in the square you leave it one